docs/azure_jumpstart_hcibox/SQLMI/_index.md

```diff
@@ -0,0 +1,103 @@
+---
+type: docs
+weight: 100
+toc_hide: true
+---
+
+# Jumpstart HCIBox - Azure Arc-enabled SQL Managed Instance
+
+## Azure Arc-enabled SQL Managed Instance on Azure Stack HCI
+
+Azure Stack HCI can provide host infrastructure for [Azure Kubernetes Service clusters](https://learn.microsoft.com/azure-stack/aks-hci/) which can be used to deploy Azure Arc-enabled SQL Managed Instance. HCIBox comes pre-configured with Azure Kubernetes Service on HCI (AKS-HCI) and an Azure Arc-enabled SQL Managed Instance. To see this in action, navigate to the resource group where you deployed HCIBox and follow these steps.
+
+- You will first need to get the Azure Custom Location Resource Provider (RP) Object ID (OID). This is required to enable [Custom Location](https://learn.microsoft.com/azure/azure-arc/platform/conceptual-custom-locations) on your Arc-enabled Kubernetes cluster.
+
+  > **NOTE: You need permissions to list all the service principals.**
```

Comment:
Can we include the specific AAD permission needed here? Global admin or Security REader I believe?